assets/helpers/contributions.js

```diff
@@ -21,43 +26,45 @@ export type Amounts = {
   oneOff: Amount,
 };
 
+export type validatedContrib = {
+  amount: number,
+  error: ContribError,
+};
+
 
 // ----- Setup ----- //
 
-const limits = {
+const LIMITS = {
   max: 2000,
   min: {
-    monthly: 5,
-    oneOff: 1,
+    RECURRING: 5,
+    ONE_OFF: 1,
   },
 };
 
+const DEFAULT_AMOUNTS = {
+  RECURRING: 10,
+  ONE_OFF: 50,
+};
 
-// ----- Functions ----- //
 
-function validate(amount: Amount, contrib: Contrib): ?ContribError {
+// ----- Functions ----- //
 
-  if (amount.value === '') {
-    return 'invalidEntry';
-  }
+export default function validate(input: string, contrib: Contrib) {
```

Comment:
* It would be nice to add the return type.
* I would change the name of this function to `parseAndValidate` or if you want to keep the name `validate` I would change the function to only return something like:  `type Validation = 'INVALID_ENTRY' | 'TOO_MUCH' | 'TOO_LITTLE' | 'VALID_ENTRY';`